kafka/vendor/six.py

```diff
@@ -71,7 +72,9 @@ def __len__(self):
             # 64-bit
             MAXSIZE = int((1 << 63) - 1)
 
-        # Don't del it here, cause with gc disabled this "leaks" to garbage
+        # Don't del it here, cause with gc disabled this "leaks" to garbage.
+        # Note: This is a kafka-python customization, details at:
+        # https://github.com/dpkp/kafka-python/pull/979#discussion_r100403389
```

Comment:
fwiw -- if upstream doesn't want to merge this change, I don't think we should keep it.

Reply:
Agreed. I left a comment on the upstream PR to bump it so we'll see what happens...